Cargo.toml

```diff
@@ -24,3 +24,7 @@ tokio = "1.29.1"
 
 [package.metadata.docs.rs]
 rustdoc-args = ["--cfg", "docs_rs"]
+
+[patch.crates-io]
+kube-client = { git = "https://github.com/MaterializeInc/kube-rs.git", rev = "838a7981483005e1af7c1338c48a4d0540a33ce4" }
```

Comment:
no, we should leave this out here - the whole point of the `[patch]` section in Cargo.toml is that it can patch versions of indirect dependencies, so even if we specify a real released version here, we can still patch it in the cloud repo to avoid the duplicate dep there.

Reply:
Okay, that just means we'll need to whitelist the `base64` duplication in `deny.toml` here, but should be able to avoid that in the cloud repo by patching this package